### DISCUSSION TITLE
I am completely new to hacktoberfest

### DISCUSSION THREAD

**Discussion by soham1826** [EddieHubCommunity/support#5526] [First-Timers]
Hello , I am completely new to hacktoberfest and open-source  how should I proceed ,???

**Comment by RaheemAmer** [comment]
## Here is what to do

1. Head to https://hacktoberfest.com/
2. Sign up if you don't have an account already
3. Once you finish, you will be redirected to a page similar to this

![image](https://user-images.githubusercontent.com/18606136/193397971-a2458461-1542-46d9-a07c-fd2d01f3186b.png)

4. Click on edit info, choose your role, and hit the save button

![image](https://user-images.githubusercontent.com/18606136/193397767-b0be233a-f9ba-4fe4-bf7a-73fc25fd637c.png)

5. How to participate?

![image](https://user-images.githubusercontent.com/18606136/193397876-08306c8a-178e-4ca0-b964-4599a77f4488.png)

6. Maintainer's role

![image](https://user-images.githubusercontent.com/18606136/193397952-c2500281-07d8-43b9-a160-70834db0f9e1.png)

7. Contributor's role

![image](https://user-images.githubusercontent.com/18606136/193398048-0341e4c6-ce0e-46b4-a2e0-cd2dad2acc30.png)

8. Any additional info can be found on the main website https://hacktoberfest.com/participation/

9. Repos to start contributing to can be found here https://github.com/topics/hacktoberfest

10. You can watch this video to know more about hacktoberfest https://www.youtube.com/watch?v=1DCCbuyF20E

# Happy hacktoberfest ❤️ and good luck

**Reply by soham1826** [reply]
Thank you 😊
